crates/integration_test/tests.sh

```diff
@@ -1,37 +1,46 @@
 #!/bin/bash
-cd ../../
-./build.sh --release
-cp ./youki ./crates/integration_test/youki
-cp ./youki_integration_test ./crates/integration_test/youki_integration_test
-cp ./runtimetest_tool ./crates/integration_test/runtimetest
-cd ./crates/integration_test
 
-RUNTIME=./youki
+set -e
 
 # syntax is 
-# test.sh build
-# test.sh run
-# test.sh run runtime-name
 # test.sh runtime-name
 
-if [[ "$1" = "build" ]]; then
-    exit 0
+if test "$(id -u)" != 0; then
+	echo "Please run as root"
+	exit 1
+fi
+
+PROJECT_ROOT=$(git rev-parse --show-toplevel)
+LOGFILE="$PROJECT_ROOT/crates/integration_test/test_log.log"
+YOUKI_INTEGRATION_TEST=$PROJECT_ROOT/youki_integration_test
+RUNTIMETEST_TOOL=$PROJECT_ROOT/runtimetest_tool
+
+if [ ! -e $YOUKI_INTEGRATION_TEST ]; then
+    echo "$YOUKI_INTEGRATION_TEST was not found, please try to build."
+    exit 1
+fi
+
+if [ ! -e $RUNTIMETEST_TOOL ]; then
+    echo "$RUNTIMETEST_TOOL was not found, please try to build."
+    exit 1
```

Comment:
Should we not add a step to run the build.sh as we originally had at the start of this? Otherwise we will need to run that script separately from this.

Reply:
Thanks for your comment. Certainly, that process is tedious, isn't it? I will change this PR to only improve to run the script anywhere. Please be patient.
However, if you want to build before you execute the integration test, we have to consider to introduce Makefile.

Reply:
Yes...
>Certainly, that process is tedious, isn't it?

Sorry :bowing_man: :bow: 
>I will change this PR to only improve to run the script anywhere. Please be patient.